src/FSharpSource.targets

```diff
@@ -758,7 +757,7 @@
     </CreateItem>
   </Target>
 
-  <Import Project="..\lkg\FSharp-$(LkgVersion)\bin\FSharp.SRGen.targets" />
+  <Import Project="..\packages\FSharp.SRGen.Build.Tasks.2.0.0\content\FSharp.SRGen.targets" />
```

Comment:
Can you add this path as well to the above properties? since we are using that group to keep track of all nuget versions.